Project_1/Styles.css

```
@@ -1,76 +1,150 @@
-img {
-    src: "Octopus.jpg";
-    max-width: 200%;
-    height: 700px;
-    width: 110%;
-    float: left;
+/* ==========
+GENERAL
+=============*/
+
+*:before,
```

This ruleset is missing one extra selector:
```suggestion
*,
*:before,
```